File: Nível 1/1072 - Intervalo 2/main.py
valor = int(input("A: "))

isin = 0
isout = 0

for i in range(1, valor + 1):
    numero = int(input("B: "))

    if 10 <= numero <= 20:
        isin += 1
    else:
        isout += 1

print(f"{isin} in")
print(f"{isout} out")

File: Nível 1/1072 - Intervalo 2/main.lua
valor = tonumber(io.read())

isIn = 0
isOut = 0

for i = 1, valor do 
    numero = tonumber(io.read())

    if numero >= 10 and numero <= 20 then
        isIn = isIn + 1
    else
        isOut = isOut + 1
    end
end

print(string.format("%d in", isIn))
print(string.format("%d out", isOut))